packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -708,7 +708,7 @@ export class Cluster extends Resource implements ICluster {
    * @param id the id of this service account
    * @param options service account options
    */
-  public addServiceAccount(id: string, options: ServiceAccountOptions) {
+  public addServiceAccount(id: string, options?: ServiceAccountOptions) {
```

Comment:
Always prefer this style to avoid null derefs:

```suggestion
  public addServiceAccount(id: string, options: ServiceAccountOptions = { }) {
```